src/main/java/games/strategy/engine/data/GameParser.java

```diff
@@ -443,15 +444,17 @@ private Object getInstance(final String className) throws GameParseException {
     Object instance = null;
     try {
       final Class<?> instanceClass = Class.forName(className);
-      instance = instanceClass.newInstance();
+      instance = instanceClass.getDeclaredConstructor().newInstance();
       // a lot can go wrong, the following list is just a subset of potential pitfalls
-    } catch (final ClassNotFoundException cnfe) {
+    } catch (final ClassNotFoundException e) {
       throw new GameParseException(mapName, "Class <" + className + "> could not be found.");
-    } catch (final InstantiationException ie) {
+    } catch (final InstantiationException e) {
       throw new GameParseException(mapName,
-          "Class <" + className + "> could not be instantiated. ->" + ie.getMessage());
-    } catch (final IllegalAccessException iae) {
-      throw new GameParseException(mapName, "Constructor could not be accessed ->" + iae.getMessage());
+          "Class <" + className + "> could not be instantiated. ->" + e.getMessage());
+    } catch (final IllegalAccessException e) {
+      throw new GameParseException(mapName, "Constructor could not be accessed ->" + e.getMessage());
+    } catch (NoSuchMethodException | SecurityException | IllegalArgumentException | InvocationTargetException e) {
```

Comment:
Sounds good, @RoiEXLab!

> All of those checked exceptions are actually thrown.

Correct.  That's why I was suggesting to simply collapse down to a single catch block for `ReflectiveOperationException` since all the checked exceptions are descendants of that class.  I'm not sure if handling the unchecked exceptions, `SecurityException` and `IllegalArgumentException`, is necessary per my previous comment.